### ISSUE TITLE
Import error

### ISSUE THREAD

**Issue by kilabyte** [petercunha/Pine#13]
Going through the dependencies i can't get past this one:

```
import win32gui, win32ui, win32con, win32api
ImportError: DLL load failed: The specified module could not be found.
```

Any ideas?

**Comment by petercunha** [comment]
Shit, sorry. I forgot to add some install steps to the readme. I'll do it this weekend.

Basically you need to install this package: https://sourceforge.net/projects/pywin32
Then python should be able to import the required packages.

Let me know if it works for you :+1:

**Comment by petercunha** [comment]
Install instructions should be fixed in 8a9fffa6bf1a50b633d73cf88c3e61596936c4c7
